content/guides/exporters/supported-exporters/Go/ApplicationInsights.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Azure Monitor"
+date: 2018-10-15
+draft: false
+weight: 3
+class: "resized-logo"
+aliases: [/supported-exporters/go/applicationinsights]
+logo: /img/partners/microsoft_logo.svg
+---
+
+- [Introduction](#introduction)
+- [Creating the exporter](#creating-the-exporter)
+- [Viewing your traces](#viewing-your-traces)
+
+## Introduction
+Azure Monitor is an extensible Application Performance Management (APM) service for web developers on multiple platforms. Offered by Microsoft Azure, it's a complete at-scale telemetry and monitoring solution.
+
+If you don't have an Azure account yet, [click here](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-overview) to start.
+
+## Creating the exporter
+Azure Monitor does not need a dedicated exporter to work with OpenCensus. Instead, it uses the readily available default exporter along with a dedicated agent known as Local Forwarder. 
+
+To learn about Local Forwarder and how to set it up, visit [this link](https://docs.microsoft.com/en-us/azure/application-insights/opencensus-local-forwarder).
+
+Here's an example of setting things up on the OpenCensus side (see [Local Forwarder repo](https://github.com/Microsoft/ApplicationInsights-LocalForwarder/blob/master/examples/opencensus/go-app/main.go) for the most up-to-date example):
+
+{{<highlight go>}}
```

Comment:
I believe the right way to highlight go is "```go"

```go
package main
import (
	"fmt"
```

Reply:
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting

Reply:
@lmolkova either works :)

Reply:
as this is Hugo and it allows for tabs hence the `highlight` sections :)

Reply:
oh, i see. If both work - can we use github compatible syntax then?

Reply:
I think I'll leave it as-is for now, since it's like this in all the other places. We'll do a separate PR if we rethink the approach.